resources/assets/js/multilingual.js

```diff
@@ -25,54 +25,43 @@
  *
  */
 ;( function( $, window, document, undefined ) {
-
```

Comment:
Please refrain from making these arbitrary whitespace changes.  I'm sure it was likely some auto-formatting from your editor, but you need to remain aware of what you're committing.  These changes provide no benefit at best, and actually break our code style at worst